src/main/java/com/google/api/codegen/metacode/InitFieldConfig.java

```diff
@@ -67,4 +69,9 @@ public boolean isFormattedConfig() {
   public boolean hasFormattedInitValue() {
     return entityName() != null && value() != null;
   }
+
+  private static String parseValueString(String valueString, String stringToHash) {
+    String randomValue = Integer.toString(Math.abs(stringToHash.hashCode()));
+    return valueString.replace(randomValueToken, randomValue);
```

Comment:
We are generating a random number every time, even if we hardly ever do the replacement. Maybe we should do a find first?